pom.xml

```diff
@@ -237,7 +237,7 @@ xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xs
     <maven-site-plugin.version>3.9.1</maven-site-plugin.version>
     <checkstyle.version>8.29</checkstyle.version>
     <surefire.fork.timeout>1200</surefire.fork.timeout>
-    <aws-java-sdk.version>1.11.1034</aws-java-sdk.version>
+    <aws-java-sdk.version>1.12.122</aws-java-sdk.version>
```

Comment:
Guess we can upgrade to 124 as well right?
https://repo1.maven.org/maven2/com/amazonaws/aws-java-sdk/1.12.124/

Reply:
Sure.  Note: 124 is newer than the PR.

Reply:
hmm, my bad. AWS folks are to be blamed they seems to be really very quick with releases. :-)